cypress/fixtures/tests/categorization/default-test-config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "source_folder": "examples/good/",
```

Comment:
A set of variant test configuration files will need to be created for the initial form of our overall test 'suite' for this app (and others).

This can be composed of the various pairings that @surchs created for our initial manual testing of the annotation tool.

Reply:
Totally, re: separate PR.